micrometer/metrics/pom.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>helidon-project</artifactId>
+        <groupId>io.helidon</groupId>
+        <version>2.1.1-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>io.helidon.microprofile.metrics</groupId>
```

Comment:
The group ID an artifact ID do not match the location. Also this module seems to be empty?

Reply:
The PR is a draft and the MP parts are not pushed yet. I pushed this file prematurely.

Reply:
I incorrectly added this.